scala/wdl4s/src/main/scala/wdl4s/RuntimeAttributes.scala

```diff
@@ -11,15 +11,15 @@ import scala.language.postfixOps
 import scalaz.Scalaz._
 import scalaz._
 
-case class RuntimeAttributes(attrs: Map[String, String])
+case class RuntimeAttributes(attrs: Map[String, Seq[String]])
 
 object RuntimeAttributes {
   def apply(ast: Ast): RuntimeAttributes = {
     val asts = ast.findAsts(AstNodeName.Runtime)
     if (asts.size > 1) throw new UnsupportedOperationException("Only one runtime block may be defined per task")
     val astList = asts.headOption map { _.getAttribute("map").asInstanceOf[AstList] }
-    val attrMap = astList map processRuntimeAttributes getOrElse Map.empty[String, String]
-    attrMap.get("memory") foreach { validateMemory }
+    val attrMap = astList map processRuntimeAttributes getOrElse Map.empty[String, Seq[String]]
+    attrMap.get("memory") flatMap { _.headOption } foreach validateMemory 
```

Comment:
I guess this is probably just "what is was before" - but any idea why memory is singled out here?